examples/atomic-registry/systemd/install.sh

```diff
@@ -1,66 +1,109 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -o pipefail
 
 INSTALL_HOST=${1:-`hostname`}
+
+# Defaults
+REGISTRYPORT=${REGISTRYPORT:-5000}
```

Comment:
It is custom when internalizing and defaulting values to use `$varname` lower case for script-local variables, keeping `$VARNAME` upper case for external or exported variables. See [`hack/test-go.sh`](https://github.com/openshift/origin/blob/master/hack/test-go.sh#L70-L79)